taurus/demo/cake.py

```diff
@@ -925,41 +925,43 @@ def find_name(name, material):
 
 
 if __name__ == "__main__":
+    encoder = TaurusJson()
     cake = make_cake(seed=42)
 
     with open("example_taurus_material_history.json", "w") as f:
         context_list = complete_material_history(cake)
         f.write(json.dumps(context_list, indent=2))
 
     with open("example_taurus_material_template.json", "w") as f:
-        f.write(thin_dumps(cake.template, indent=2))
+        f.write(encoder.thin_dumps(cake.template, indent=2))
```

Comment:
Are we still using these example dumps elsewhere?  I thought they were feeding a platform test or development target.

Reply:
Yes, when we make data model changes we use this to update some tests in platform-backend.